cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/preferences.rst

```diff
@@ -0,0 +1,163 @@
+.. meta::
+:author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+      :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _http-restful-api-preferences:
+.. _http-restful-api-v3-namespace:
+
+===========================================================
+Preferences HTTP RESTful API
+===========================================================
+
+.. highlight:: console
+
+Use the CDAP Preferences HTTP API to save, retrieve, delete preferences in CDAP.
```

Comment:
Use the CDAP Preferences HTTP RESTful API to save, retrieve, and delete preferences in CDAP.